server/api.js

```diff
@@ -3,16 +3,74 @@ import { Router } from "express";
 
 import { Connection } from "./db";
 
+
+import { AuthorizationCode } from "simple-oauth2";
+
 const router = new Router();
+const { Octokit } = require("@octokit/core");
+
+
+const client = new AuthorizationCode({
+	client: {
+		//these would come from the github where the app is registered.
+		id: process.env.CLIENT_ID,
+		secret: process.env.CLIENT_SECRET,
+	},
+	auth: {
+		tokenHost: 'https://github.com',
+		tokenPath: '/login/oauth/access_token',
+		authorizePath: '/login/oauth/authorize',
+	},
+});
+
+const authorizationUri = client.authorizeURL({
+	//we can put in the redirect_uri when we deploy the app
+	redirect_uri: 'http://localhost:3100/api/callback',
```

Comment:
would be nice to store in the process.env, but it's ok for now

Reply:
Yeah, that would be better